script/scriptComplexity.js

```diff
@@ -0,0 +1,21 @@
+
+// used the bubble algorithm
+function sortArray(arr) {
+    let temp;
+    for (let i = 0; i < arr.length - 1; i++) {
+        for (let j = 0; j < arr.length - i - 1; j++) {
+            if (arr[j] > arr[j + 1]) {
+                temp = arr[j];
```

Comment:
No way, this is not the solution !!!

Reply:
i have updated the solution to work with count sort, (as shown in the class on Monday).
but i think that this solution can have unseen repercussions, i have added an array where this solution is not a good one
`let arr2 = [10, 4, 30, 1728, 9, 17, 2, 24, 11, 23, 5, 7];`
here there is the number 1728 it is equal to 12^3 and it is equal to n^3
that means that the complexity of the second loop is O(n^3)
if there was a limit for the numbers in the array, for example an array of students grades that are from 0 to 100,
then this solution will be perfect.

Reply:
sorry for the long comment  😅